web/cobrands/fixmystreet/offline.js

```diff
@@ -419,9 +419,18 @@ if ($('#offline_list').length) {
 } else {
     fixmystreet.offlineBanner.make(false);
 
-    // On /my/planned, when online, cache all shortlisted
+    // On /my/planned, when online, cache all shortlisted automatically on mobile.
+    // On desktop, show a button to cache them.
     if (location.pathname === '/my/planned') {
-        fixmystreet.offline.updateCachedReports();
+        if ($("html").hasClass("mobile")) {
+            $(".js-cache-reports").hide();
+            fixmystreet.offline.updateCachedReports();
+        } else {
+            $(".js-cache-reports").on("click", function() {
+                $(".js-cache-reports").hide();
```

Comment:
Think this could have a preventDefault, just to stop # in the URL?

Reply:
Also this hides the link but keeps the bottom bar, should we hide the whole div?